WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/BlockListItem.kt

```diff
@@ -233,7 +233,8 @@ sealed class BlockListItem(val type: Type) {
     data class LoadingItem(val loadMore: () -> Unit, val isLoading: Boolean = false) : BlockListItem(LOADING_ITEM)
 
     data class ActivityItem(val blocks: List<Block>) : BlockListItem(ACTIVITY_ITEM) {
-        data class Block(val label: String, val boxes: List<Box>)
+        data class Block(val label: String, val boxItems: List<BoxItem>)
+        data class BoxItem(val box: Box, val contentDescription: String? = null)
```

Comment:
Let me know what you think about the naming here. I am not sure if I am adhering to the naming convention being used so I am rethinking it. So the BoxItem could be renamed to `Box` and the enum `BoxStyle` or `BoxType`

Reply:
Okay thanks @planarvoid 👌